resolwe/permissions/tests/test_data.py

```diff
@@ -62,9 +62,9 @@ def test_post(self):
         d = Data.objects.get(pk=resp.data['id'])
         self.assertTrue(datetime.now() - d.modified < timedelta(seconds=1))
         self.assertTrue(datetime.now() - d.created < timedelta(seconds=1))
-        self.assertEqual(d.status, 'RE')
-        self.assertEqual(d.started, None)
-        self.assertEqual(d.finished, None)
+        self.assertEqual(d.status, 'OK')
+        self.assertTrue(datetime.now() - d.started < timedelta(seconds=1))
```

Comment:
We should stop using just `datetime.now`. Read about [Django's Time Zones](https://docs.djangoproject.com/en/1.9/topics/i18n/timezones/). We should make resolwe invariant to Django settings and when time zone support not enabled use:
```
import datetime
now = datetime.datetime.now()
```
but when time zone support is enabled (USE_TZ=True) use:
```
from django.utils import timezone
now = timezone.now()
```